### DISCUSSION TITLE
Question about the calculations of the fraction X, Y, Z for predicting the tissue plasma partition coefficient by Rodgers & Rowland method

### DISCUSSION THREAD

**Discussion by jessechao830330** [Open-Systems-Pharmacology/Forum#990] [Questions & Problems] [answered]
Hello team,

When I dig into all the parameters calculated in the PK-Sim by Mo-Bi and I found out that the fraction X, Y, Z used to calculate the partition coefficient by the Rodgers & Rowland method have a suspected mistake in implementing the physiological parameters. 

![Screenshot 2021-10-14 at 00 08 22](https://user-images.githubusercontent.com/66793768/137221618-e01c88c2-f13b-49d5-97b5-210e01f629d5.png)

The question I have is about the parameter pH_IW. Since in the physiological data we have all the intracellular pH for all organs, why the calculation for the fraction X is using a set value (7.0) for  pH_IW from the "Organism" for all organs, instead of using the pH (intracellular) from each organ individually? Is it for faster simulation? or I have some misunderstandings about the equation?

It will be great if you could give me some feedback.  Thank you very much.

Have a nice day.

Best,
Jesse

**Comment by prvmalik** [comment, marked as answer]
In the original publications (2005, 2006) with the associated evaluation, the Rodgers and Rowland method uses pH_IW 7.0 for solid organs and pH_IW 7.22 for blood cells. In PKSim, it has been employed as published and evaluated.

You could change the equation to use the pH_IW values from the 2008 toxicology paper, but the method may need to be re-evaluated with a suitable list of compounds, and any changes to the algorithm would have to be disclosed on academic or regulatory documents, if you are claiming that you are using the algorithm.

**Reply by jessechao830330** [reply]
Hello Paul,
Thank you very much for your clear explanation.
